src/symbolize/gimli.rs

```diff
@@ -613,7 +613,18 @@ pub unsafe fn resolve(what: ResolveWhat<'_>, cb: &mut dyn FnMut(&super::Symbol))
                 });
             }
         }
-
+        if let Some((object_cx, object_addr)) = cx.object.search_object_map(addr as u64) {
```

Comment:
I played around with this a bit locally, ideally we'd deduplciate this and the above `find_frames` loop since I think they're basically identical. Unfortunately I kept running into lifetime errors. It may mean that the lazily-populated map for objects should perhaps use interior mutability, or ideally something like `LazyCell`, although that's not stable yet.

Reply:
I wasn't able to come up with anything for this that felt like a net decrease in complexity.